### DISCUSSION TITLE
Ability to copy only automation page

### DISCUSSION THREAD

**Discussion by binjac** [SynthstromAudible/DelugeFirmware#2098] [Ideas]
I often have fun with the filter automation on 16 steps grid but when i want to copy this 16 steps automation to the rest of a longer loop page by page the Learn + <> copies the notes as well, I would love a shortcut or method to be able to copy/paste only automation when in clip automation view

**Comment by seangoodvibes** [comment]
I think you're doing something wrong or this is a bug and an issue should be opened up. It should only be copying automation if you're in automation view.

The shortcut for copying automation is not learn + <>, it's learn + press gold knob 

Pasting automation is shift + learn + press gold knob

**Reply by binjac** [reply]
Thank you, I need to update my cheatsheets! Indeed wrong shortcut...
